src/main/java/com/fnp/Collections/sort-using-comparator/employeestorage/Employee.java

```diff
@@ -0,0 +1,61 @@
+package employeestorage;
+
+public class Employee implements Comparable<Employee>{
+
+	int id;
+	String name;
+	String dept;
+	long phone;
+	float sal;
+	
+	public int getId(){
+			return id;
+	}
+	
+	public void setId(int id){
+			this.id = id; 
+	}
+	public String getName(){
+			return name;
+	}
+	
+	public void setName(String name){
+			this.name = name; 
+	}
+	
+	public String getDept(){
+			return dept;
+	}
+	
+	public void setDept(String dept){
+			this.dept = dept; 
+	}
+	
+	public long getPhone(){
+			return phone;
+	}
+	
+	public void setPhone(long phone){
+			this.phone = phone; 
+	}
+	
+	public float getSal(){
+			return sal;
+	}
+	
+	public void setSal(float sal){
+			this.sal = sal; 
+	}
+	
+	public int compareTo(Employee employee){
+		if(id>employee.id)
```

Comment:
there should be space between operator and variable which is being checked on right and left..ignore for now..take care in later commits